hw/ip/flash_ctrl/doc/_index.md

```diff
@@ -173,6 +173,15 @@ When an RMA entry request is received from the life cycle manager, the flash con
 The flash controller then initiates RMA entry process and notifies the life cycle controller when it is complete.
 Unlike the seed phase, after the RMA phase, the flash controller does not grant control back to software as the system is expected to reboot after an RMA attempt.
 
+#### RMA Entry
+During RMA entry, the flash controller "wipes" the contents of the following
+- creator partition
+- owner partition
+- Isolated partition
+- All data partitions
+
+This process ensures that after RMA there is no sensitive information left that can be made use of the tester.
```

Comment:
nit: ... can be made use of **on** the tester?

Reply:
fixed, thanks!